Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM alpine:3 AS builder
```

Comment:
I think you need to stick to specific version of `alpine`, not just a major one because of the packages versions